tests/nfv/prepare_env.pm

```diff
@@ -64,22 +73,42 @@ sub run {
     else {
         die "OS VERSION not supported. Available only on 15 and 12-SP4";
     }
-    assert_script_run("cd /root/vswitchperf/src/trex; make");
 
-    # Setup ssh keys, wait until trafficgen machine is up and ready
-    mutex_wait('NFV_trafficgen_ready', $child_id);
+    record_info("Clone Trex repo inside VSPerf directories");
+    assert_script_run("cd /root/vswitchperf/src/trex; make", timeout => 200);
+
+    record_info("Copy configuration files");
+    assert_script_run("curl " . data_url('nfv/conf/00_common.conf') . " -o /root/vswitchperf/conf/00_common.conf");
+    assert_script_run("curl " . data_url('nfv/conf/01_testcases.conf') . " -o /root/vswitchperf/conf/01_testcases.conf");
+    assert_script_run("curl " . data_url('nfv/conf/02_vswitch.conf') . " -o /root/vswitchperf/conf/02_vswitch.conf");
+    assert_script_run("curl " . data_url('nfv/conf/03_traffic.conf') . " -o /root/vswitchperf/conf/03_traffic.conf");
+    assert_script_run("curl " . data_url('nfv/conf/04_vnf.conf') . " -o /root/vswitchperf/conf/04_vnf.conf");
+    assert_script_run("curl " . data_url('nfv/conf/05_collector.conf') . " -o /root/vswitchperf/conf/05_collector.conf");
+    assert_script_run("curl " . data_url('nfv/conf/06_pktfwd.conf') . " -o /root/vswitchperf/conf/06_pktfwd.conf");
+    assert_script_run("curl " . data_url('nfv/conf/07_loadgen.conf') . " -o /root/vswitchperf/conf/07_loadgen.conf");
+    assert_script_run("curl " . data_url('nfv/conf/08_llcmanagement.conf') . " -o /root/vswitchperf/conf/08_llcmanagement.conf");
+    assert_script_run("curl " . data_url('nfv/conf/10_custom.conf') . " -o /root/vswitchperf/conf/10_custom.conf");
+    assert_script_run("sed -i 's/trafficgen_ip/$trafficgen_ip/' -i /root/vswitchperf/conf/10_custom.conf");
+
+    record_info("Waiting for NFV_TRAFFICGEN_READY");
+    mutex_wait('NFV_TRAFFICGEN_READY', $child_id);
+
+    record_info("Generate ssh keypair and ssh-copy-id to the Traffic generator machine");
     assert_script_run('ssh-keygen -b 2048 -t rsa -q -N "" -f ~/.ssh/id_rsa');
-    exec_and_insert_password("ssh-copy-id -o StrictHostKeyChecking=no root\@$host1");
-    exec_and_insert_password("ssh-copy-id -o StrictHostKeyChecking=no root\@$host2");
+    exec_and_insert_password("ssh-copy-id -o StrictHostKeyChecking=no root\@$trafficgen_ip");
 
-    # Bring mellanox interfaces up
+    record_info("Bring mellanox interfaces up");
     assert_script_run("ip link set dev eth2 up");
     assert_script_run("ip link set dev eth2 up");
 
-    mutex_create("NFV_testing_ready");
-    wait_for_children;
+    record_info("Stop SuSEfirewall2.service");
+    script_run("systemctl stop SuSEfirewall2.service");
+
+    record_info("NFV Installation ready");
+    mutex_create("NFV_VSPERF_READY");
 }
 
+
```

Comment:
I don't see a need for a second blank line

Reply:
done